scripts/ztunnel-redirect-inpod.sh

```diff
@@ -4,46 +4,43 @@
 
 set -ex
 
-# Below is from config.sh but used in redirect-worker.sh as well
-POD_OUTBOUND=15001
-POD_INBOUND=15008
-POD_INBOUND_PLAINTEXT=15006
-
 # CONNMARK is needed to make original src work. we set conn mark on prerouting. this is will not effect connections
 # from ztunnel to outside the pod, which will go on OUTPUT chain.
 # as we are in the pod ns, we can use whatever iptables is default.
 iptables-restore --wait 10 <<EOF
-# Generated by iptables-save v1.8.9 (nf_tables) on Thu Jun 22 11:52:46 2023
 *mangle
 :PREROUTING ACCEPT [0:0]
 :INPUT ACCEPT [0:0]
 :FORWARD ACCEPT [0:0]
 :OUTPUT ACCEPT [0:0]
 :POSTROUTING ACCEPT [0:0]
-
--A PREROUTING -m mark --mark 1337/0xfff -j CONNMARK --set-xmark 0x111/0xfff
--A PREROUTING -p tcp -m tcp --dport $POD_INBOUND -m mark ! --mark 1337/0xfff -j TPROXY --on-port $POD_INBOUND --on-ip 127.0.0.1 --tproxy-mark 0x111/0xfff
--A PREROUTING -p tcp -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
--A PREROUTING ! -d 127.0.0.1/32 -p tcp -m mark ! --mark 1337/0xfff -j TPROXY --on-port $POD_INBOUND_PLAINTEXT --on-ip 127.0.0.1 --tproxy-mark 0x111/0xfff
--A OUTPUT -m connmark --mark 0x111/0xfff -j CONNMARK --restore-mark --nfmask 0xffffffff --ctmask 0xffffffff
+:ISTIO_OUTPUT - [0:0]
```

Comment:
Eventually, if we keep this around, I want to see if we can replace this with a direct invocation of `istio-iptables` so we don't have to keep these in sync.

Reply:
I generally like that, though it is tricky since today all of our repos are a DAG in dependencies. This would introduce a cycle. Not impossible, certainly